lib/modules/styleTweaks.js

```diff
@@ -406,81 +392,6 @@ modules['styleTweaks'] = {
-			if (thisExpando.classList.contains('twitterLoaded')) {
-				thisExpando.style.display = 'block';
-				return;
-			}
-			var twitterLink = e.target.previousSibling.querySelector('.title');
-			if (twitterLink) twitterLink = twitterLink.getAttribute('href').replace('/#!', '');
-			var match = twitterLink.match(/twitter.com\/[^\/]+\/(?:status|statuses)\/([\d]+)/i);
-			if (match !== null) {
-				// var jsonURL = 'http://api.twitter.com/1/statuses/show/'+match[1]+'.json';
-				var jsonURL = 'https://api.twitter.com/1/statuses/oembed.json?id=' + match[1],
-					thisJSON = {
-						requestType: 'loadTweet',
-						url: jsonURL
-					};
-				if (BrowserDetect.isChrome()) {
-					// we've got chrome, so we need to hit up the background page to do cross domain XHR
-
-					// first, we need to see if we have permissions for the twitter API...
-					var permissionsJSON = {
-						requestType: 'permissions',
-						callbackID: permissionQueue.count,
-						data: {
-							origins: ['https://api.twitter.com/*']
-						}
-					};
-					// save a function call that'll run the expando if our permissions request
-					// comes back with a result of true
-					permissionQueue.onloads[permissionQueue.count] = function(hasPermission) {
-						if (hasPermission) {
-							chrome.runtime.sendMessage(thisJSON, function(response) {
-								// send message to background.html
-								var tweet = response;
-								$(thisExpando).html(tweet.html);
-								thisExpando.style.display = 'block';
-								thisExpando.classList.add('twitterLoaded');
-							});
-						} else {
-							// close the expando since we don't have permission.
-							$(e.target).removeClass('expanded').addClass('collapsed collapsedExpando');
-						}
-					};
-					permissionQueue.count++;
-
-					// we do a noop in the callback here because we can't actually get a
-					// response - there's multiple async calls going on...
-					chrome.runtime.sendMessage(permissionsJSON, function(response) {});
-
-				} else if (BrowserDetect.isSafari()) {
-					// we've got safari, so we need to hit up the background page to do cross domain XHR
-					modules['styleTweaks'].tweetExpando = thisExpando;
-					safari.self.tab.dispatchMessage(thisJSON.requestType, thisJSON);
-				} else if (BrowserDetect.isOpera()) {
-					// we've got opera, so we need to hit up the background page to do cross domain XHR
-					modules['styleTweaks'].tweetExpando = thisExpando;
-					opera.extension.postMessage(JSON.stringify(thisJSON));
-				} else if (BrowserDetect.isFirefox()) {
-					// we've got a jetpack extension, hit up the background page...
-					// we have to omit the script tag and all of the nice formatting it brings us in Firefox
-					// because AMO does not permit externally hosted script tags being pulled in from
-					// oEmbed like this...
-					jsonURL += '&omit_script=true';
-					modules['styleTweaks'].tweetExpando = thisExpando;
-					self.postMessage(thisJSON);
```

Comment:
Well, they must not have been paying much attention, since they didn't reject [4.5.4](https://i.imgur.com/var8jVQ.png), or [4.5.3](https://i.imgur.com/GXVKzYJ.png), or [4.5.0.2](https://cloud.githubusercontent.com/assets/7673145/6791007/27204c5a-d182-11e4-8595-d6825da9bff2.png) - all of which include the script (we still shouldn't risk it for v4.6 - but it is amusing).

After some investigation: Twitter expandos were downgraded for about a year, which ended about a year ago: the [original commit](https://github.com/honestbleeps/Reddit-Enhancement-Suite/commit/d865829ccba7769703c2dfd74bc48528065b0b33) to omit the script was after release v4.2.0.2, and it was accidentally nullified by [this commit](https://github.com/mc10/Reddit-Enhancement-Suite/commit/c993360b64f5bef1b507c54dd971bf99d3efd7f5#diff-0767dec274256dd1746817f7f75c9890L15095) after release v4.3.1.2.